Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -330,7 +330,7 @@ public extension DocumentationNode {
         let taskGroups: [LinkDestinationSummary.TaskGroup]?
         if includeTaskGroups {
             switch kind {
-            case .tutorial, .tutorialArticle, .technology, .technologyOverview, .chapter, .volume, .onPageLandmark:
+            case .tutorial, .tutorialArticle, .tutorialTableOfContents, .chapter, .volume, .onPageLandmark:
```

Comment:
Is the removal of `.technologyOverview` here intended?

Reply:
It was with the thinking that we're never using `.technologyOverview` anywhere so it should be safe to remove the checks for it but now that you point it out I feel that it's better to be extra safe and add it back.

Reply:
Added back in https://github.com/swiftlang/swift-docc/pull/1039/commits/aa2479396d8f5933ce1943f40d6cfc981dbf4fd6